### PR TITLE
Messages open groups

### LINKED ISSUE #898: Pressing on text highlights message
ok normally when one double or triple clicks inside a text bubble it's in order to select all the text for copy-pasting f.ex., whereas in Session this automatically brings up the delete menu

### PR DESCRIPTION
**Pull request by vincentbavitz**
Fixes #898 and group joining on enter

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -210,7 +210,7 @@
     "node": "^10.13.0"
   },
   "build": {
-    "appId": "com.loki-project.messenger-desktop",
+    "appId": "org.getsession.session-desktop",
```
Comment (Mikunj) [inline package.json]: uhhh i wouldn't do this here and instead move it to another PR. This is because the `appId` is used as a unique app identifier on mac and windows. If we change it, it might break auto-updating or give us some errors on install. We should test it and see if any of those things happen before changing it.

Comment (vincentbavitz) [reply package.json]: Alright I'll redact
